src/python/WMCore/ReqMgr/DataStructs/Request.py

```diff
@@ -20,6 +20,9 @@
 from WMCore.ReqMgr.DataStructs.RequestStatus import REQUEST_START_STATE
 from WMCore.ReqMgr.DataStructs.RequestError import InvalidSpecParameterValue
 
+#TODO: this need to be update from DB instead of hard coding for more flexibility
+RESULT_FILTER_FOR_DAS = ["RequestName", "OutputDatasets", "InputDatasets", "PrepID"]
```

Comment:
> a) why InputDatasets (plural)? Is it for TaskChain cases? Or it adds the Pileup sample?

It is just the key, initially we are supposed to support multiple inputdatasets so that is why we have Inputdatasets altough we have only one elements,

> b) giving a second thought, OutputDatasets sounds useless, since the user using DAS will know already which dataset or block or file he wants to look at

I will remove that,

> c) PrepID, GlobalTag and CMSSWVersion can be different for TaskChain (and StepChain) requests, are you considering it?

I think DAS has different ways to get the ConfigID and PrepID.
 None of Taskchain property are considered here. Only top level. If we add TaskChain properties I will add the list instead of string.